doc/releases/changelog-dev.md

```diff
@@ -9,16 +9,12 @@
 <h4>Other Improvements</h4>
 
 * Added `qml.devices.qubit_mixed` module for mixed-state qubit device support. This module introduces:
-  - A new API for mixed-state operations
-  - An `apply_operation` helper function featuring:
-    - Two density matrix contraction methods using `einsum` and `tensordot`
-    - Optimized handling of special cases including:
-      - Diagonal operators
-      - Identity operators 
-      - CX (controlled-X)
-      - Multi-controlled X gates
-      - Grover operators
-  [(#6379)](https://github.com/PennyLaneAI/pennylane/pull/6379)
+
+  [(#6379)](https://github.com/PennyLaneAI/pennylane/pull/6379) An `apply_operation` helper function featuring:
+
+  * Two density matrix contraction methods using `einsum` and `tensordot`
+
+  * Optimized handling of special cases including: Diagonal operators, Identity operators, CX (controlled-X), Multi-controlled X gates, Grover operators
```

Comment:
This changelog entry might've been a merge artifact, but it's likely a duplicate so should be resolved.

Reply:
@mudit2812 Yeah this is from docs fixing conversation from another pr [here](https://github.com/PennyLaneAI/pennylane/pull/6503). This corrected docs also appear in my other removal pr's. Anyway the orignal docstrings are not rendered correctly and will be fixed; I can also create another PR separately to fix it. What do you think?

Reply:
Reverted